workflows/quickies/phylogenetic-context/Q1-phylogenetic-context-analysis-tests.yml

```
@@ -0,0 +1,22 @@
+- doc: Test outline for Q1-phylogenetic-context-analysis
+  job:
+    List of NCBI accessions:
+      class: File
+      path: test-data/List of NCBI accessions.txt
+      filetype: txt
+    Reference:
+      class: File
+      path: test-data/Reference.genbank
+      filetype: genbank
+    My sequences:
+      class: Collection
+      collection_type: list
+      elements:
+      - class: File
+        identifier: fri2
+        path: test-data/My sequences_fri2.fasta
+  outputs:
+    Phylogenetic tree (PDF):
+      path: test-data/Phylogenetic tree (PDF).png
```

```suggestion
      path: test-data/Phylogenetic tree (PDF).png
      compare: sim_size
      delta: 10000
```

you can use most of the directives that we also use for tool tests (https://docs.galaxyproject.org/en/latest/dev/schema.html#id106).